LCD_digits/digits/LCDNumber.js

```diff
@@ -0,0 +1,51 @@
+const LCDDigit = require("./LCDDigit").LCDDigit;
+
+class LCDNumber {
+    constructor(number) {
+        this.number = number;
+        this.lcdDigits = LCDNumber.generateLCDDigits(number);
+    }
+
+    printLine1() {
```

Comment:
These can all be changed to a map:

```javascript
//insert the correct methods printLine2 => digit.getLine2, etc
printLine1() {
        const line = this.lcdDigits.map(digit => digit.getLine1()).join(" ");
        console.log(line);
    }
```

A map is typically used if you want an output object or variable for each iteration. This seems repeated though since the method is the only thing that's changing. This might be over engineered as a result of not writing tests. It's not directly repeated but I still wouldn't call it DRY. It looks like the `print()` is the only method intended to be called so it could just call the others internally

```javascript
printLine(lineNumber) {
        const line = this.lcdDigits.map(digit => digit[`getLine${lineNumber}`]()).join(" ");
        console.log(line);
    }

    print() {
        this.printLine(1);
        this.printLine(2);
        this.printLine(3);
    }
```
but that feels kind of hacky.